README.md

```diff
@@ -128,6 +128,9 @@ Thank you so much for contributing!!
 * [__Gruvbox Dark__](themes/prism-gruvbox-dark.css) (by [Azat S.](https://github.com/azat-io))<br />
 [![Gruvbox Dark](screenshots/prism-gruvbox-dark.png)](themes/prism-gruvbox-dark.css)
 
+* [__Gruvbox Light__](themes/prism-gruvbox-light.css) (by [Michael Schnerring](https://schnerring.net))<br />
```

Comment:
Could you please link your GitHub instead?

Linking your website in the `@author` tag is fine.

Reply:
Done!